docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'placeholder',
         type: 'string',
-        href: 'basicExample',
+        description:
+          'If not provided, the first item in the list will be shown. Be sure to localize the text.',
       },
       {
         name: 'size',
         type: '"md" | "lg"',
         required: false,
-        description: 'md: 40px, lg: 48px',
+        description: `md: 40px, lg: 48px. See the [size](#Size) variant to learn more.`,
         defaultValue: 'md',
       },
       {
         name: 'value',
         type: 'string',
-        description: 'Value that is selected.',
-        href: 'basicExample',
+        description: 'The currently-selected value',
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    id="basicExample"
-    name="Example"
-    description={`Make sure to attach a \`Label\` to every SelectList.`}
-    defaultCode={`
-function Example(props) {
-  const [city, setCity] = React.useState('la')
-  const cityOptions = [
-    {
-      value: "bos",
-      label: "Boston"
-    },
-    {
-      value: "la",
-      label: "Los Angeles"
-    },
-    {
-      value: "sf",
-      label: "San Francisco"
-    }
-  ];
-  return (
-    <SelectList
-      id="city"
-      name="city"
-      onChange={({ value }) => setCity(value)}
-      options={cityOptions}
-      placeholder="Select city"
-      label="City"
-      value={city}
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use SelectList when the user needs to select from a simple list of items."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Austin', value: 'austin'},
+    {label:'Los Angeles', value: 'la'},
+    {label:'New York', value: 'nyc'},
+    {label:'San Francisco', value: 'sf'},
+    {label:'Seattle', value: 'seattle'},
+  ]}
+  label='City'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList when additional functionality such as subtext or images are needed. Use Dropdown instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'List item 1', value: 'Value'},
+    {label:'I want to put an image here, but I can’t :( ', value: 'Value'},
+    {label:'List item 3', value: 'Value'},
+    {label:'List item 4', value: 'Value'},
+    {label:'List item 5', value: 'Value'},
+  ]}
+  label=''
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Order the list items in SelectList either alphabetically or by usage."
+        defaultCode={`
+<SelectList
```

Comment:
Same here, I'd find a good case of checkboxes in Pinboard, and translate it into SelectList or a common example of misused SelectList

For example, we could do gender here.
![image](https://user-images.githubusercontent.com/10593890/110532346-b0180980-80ea-11eb-8fb7-0d2f5255bf19.png)

![image](https://user-images.githubusercontent.com/10593890/110532375-b7d7ae00-80ea-11eb-897c-0a10b95a0faa.png)